README.md

```diff
@@ -6,6 +6,12 @@ Aside from generating Yaml and running *oc* commands to apply them to your clust
 
 Helper scripts to automate installing CRC and required tools with versions used in openstack-k8s-operators can be found in [CRC/tools deployment](devsetup/README.md). These scripts/playbook required sudo permissions.
 
+**Note**
+The `install_yamls` project expects several dependencies on the host machine.
+Without them the deployment will fail and you will have install them first.
+In general terms, all tools required by Openshift are also required by `install_yamls`.
+Most importanly, the `kubectl` must be present on the system.
```

Comment:
You can also symlink `kubectl` to `oc`.  Or at least I've done that in past environments without explicitly installing `kubectl`.

Reply:
I suppose that could be alternative. But I think we should keep it simple for sake of the reader.